Gemfile

```diff
@@ -28,8 +28,7 @@ group :test do
 end
 
 platforms :rbx do
-  # TODO: What are the effects of not having this?
-  # gem 'rubysl', '~> 2.0'
+  gem 'rubysl', '~> 2.0'
```

Comment:
@maddymarkovitz What ended up being the issue with this?

Reply:
I'm really not sure. Early on in the process @oeoeaio and I found that you had to remove the `rubysl` dependency to get the gems to install. I don't understand why, because we weren't targeting the `rbx` (Rubinius?) platform.
A separate question is why this is in here at all if Rubininus isn't officially supported.

Reply:
I vote we just drop this. This was [added](https://github.com/que-rb/que/pull/21) back in 2014 in `0.6.0` when the test suite was running in TravisCI.